### PR TITLE
update virtual study model

### PR DESCRIPTION
**Pull request by kalletlak**

### CODE REVIEW

**Comment by jjgao** [comment]
@zhx828 we hope to get this one out before the 1.10 release.

@kalletlak does this pr depend on this one: https://github.com/cBioPortal/session-service/pull/14?

**Comment by kalletlak** [comment]
@jjgao yes!
And https://github.com/cBioPortal/cbioportal/pull/3343 depends on this and https://github.com/cBioPortal/session-service/pull/14

**Comment by jjgao** [comment]
Does cBioPortal/cbioportal#3343  depend on this pr?

File: app/scripts/views/components/shareVirtualStudy/shareVirtualStudy.js [hunk 1]
```diff
@@ -154,18 +154,18 @@
...
                       var _selectedSamplesNum = 0;
                       var _selectedPatientsNum = 0;
-                      _.each(self_.stats.selectedCases, function (studyCasesMap) {
+                      _.each(self_.stats.studies, function (studyCasesMap) {
                         _selectedSamplesNum += studyCasesMap.samples.length;
-                        _selectedPatientsNum += studyCasesMap.patients.length;
                       });
                       self_.selectedSamplesNum = _selectedSamplesNum;
-                      self_.selectedPatientsNum = _selectedPatientsNum;
+                      self_.selectedPatientsNum = _selectedSamplesNum;
```
Comment (zhx828) [inline app/scripts/views/components/shareVirtualStudy/shareVirtualStudy.js]: # of patients always matches with # of samples?

File: app/scripts/views/components/shareVirtualStudy/shareVirtualStudy.js [hunk 2]
```diff
@@ -154,18 +154,18 @@
...
 
+                      console.log(self_.stats)
```
Comment (zhx828) [inline app/scripts/views/components/shareVirtualStudy/shareVirtualStudy.js]: No longer needed.

File: app/scripts/views/components/shareVirtualStudy/shareVirtualStudy.js (hunk 1 above)
Comment (kalletlak) [reply app/scripts/views/components/shareVirtualStudy/shareVirtualStudy.js]: Sorry! by mistake changed it.
 btw where are `selectedSamplesNum` and `selectedPatientsNum` used here?

**Review by zhx828 — APPROVED** [review]

**Pull request merged**